hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/blockmanagement/TestDatanodeManager.java

```diff
@@ -978,9 +978,9 @@ public void testRemoveIncludedNode() throws IOException {
    * @throws IOException
    */
   private void verifyPendingRecoveryTasks(
-      int numReplicationBlocks, int numECBlocks,
-      int maxTransfers, int maxTransfersHardLimit,
-      int numReplicationTasks, int numECTasks, boolean isDecommissioning)
+      int numReplicationBlocks, int numEcReplicatedBlocks, int numECBlocks,
+      int maxTransfers, int maxTransfersHardLimit, int numReplicationTasks,
+      int numECReplicatedTasks, int numECTasks, boolean isDecommissioning)
```

Comment:
- Please update the javadoc comments for the new variables.
- The names of `numECBlocks` and `numECTasks` should be refactored, as I suggested in other places.